docs/user-guide/consuming.rst

```diff
@@ -197,6 +197,40 @@ consumers to use if they need configuration options. Refer to the
 :ref:`conf-consumer-config` in the Configuration documentation for details.
 
 
+Monitoring
+==========
+
+A Fedora Messaging consumer can start a dedicated HTTP server to let users
+monitor its state. You can configure the port in the configuration file under
+the ``[monitoring]`` section, with ``address`` and ``port``.
+The default address is empty, which means that the monitoring server will
+listen on all interfaces. The default port is ``0``, which means that the
+monitoring server is disabled.
+
+When a consumer is running with the monitoring server enabled, you can get the
+following data::
+
+    $ curl http://localhost:8070/live
+    {"status": "OK"}
```

Comment:
It'd be nice to include a bit more documentation on this endpoint to indicate what set of values it returns. Looking at the code it seems it only ever returns this, so just calling that out explicitly in the docs would be good.